tests/ssg_test_suite/rule.py

```diff
@@ -292,17 +324,14 @@ def _get_scenarios(self, rule_dir, scripts, scenarios_regex, benchmark_cpes):
 
         return scenarios
 
-    def _check_rule(self, rule, remote_dir, state, remediation_available):
+    def _check_rule(self, rule, scenarios, remote_dir, state, remediation_available):
```

Comment:
`CombinedChecker` class in `ssg_test_suite/combined.py` inherits from `RuleChecker` and uses this `_check_rule` method. Please, fix the method usage there.

Reply:
Good catch, this is a non-trivial finding, and I have done my best to make the `RuleChecker` more extension-friendly, so the `CombinedChecker` doesn't contain duplicated code.